pages/osx/valet.md

```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on http://<project>.test
```

Comment:
For examples urls/hostnames we usually use example.x. Something like 
```suggestion
> A minimalist Laravel development environment that allows hosting sites via local tunnels on http://<example>.test
```
Do you think it makes sense to change it to that?

Reply:
done